languages/minecraft/acf-minecraft_de.properties

```diff
@@ -25,13 +25,14 @@ acf-minecraft.invalid_world = Fehler: Diese Welt gibt es nicht.
 acf-minecraft.you_must_be_holding_item = Fehler: Du musst einen Gegenstand in deiner Haupthand halten.
 acf-minecraft.player_is_vanished_confirm = \
     Warnung: <c2>{vanished}</c2> ist getarnt. Lasse die Tarnung nicht auffliegen!\n\
-    Um die Aktion zu best�tigen, h�nge <c2>:confirm</c2> hinten an den Namen an.\n\
+    Um die Aktion zu bestätigen, hänge <c2>:confirm</c2> hinten an den Namen an.\n\
     Bsp: <c2>{vanished}:confirm</c2>
-acf-minecraft.username_too_short = Fehler: Spielername zu kurz, er musst mindestens drei Buchstaben lang sein.
-acf-minecraft.is_not_a_valid_name = Fehler: <c2>{name}</c2> ist kein valider Spielername.
-acf-minecraft.multiple_players_match = Fehler: Es gibt mehrere Spieler deren Namen <c2>{search}</c2> <c3>({all})</c3> �hneln, du musst genauer sein.
-acf-minecraft.no_player_found_server = Es gibt keinen Spieler auf diesem Server dessen Namen <c2>{search}</c2> �hnelt.
-acf-minecraft.no_player_found = Es gibt keinen Spieler dessen Namen <c2>{search}</c2> �hnelt.
-acf-minecraft.location_please_specify_world=Fehler: Bitte die Welt angeben. Beispiel: <c2>world:x,y,z</c2>.
-acf-minecraft.location_please_specify_xyz=Fehler: Bitte x, y, z. Beispiel: <c2> Welt: x, y, z </ c2>.
-acf-minecraft.location_console_not_relative=Fehler: Konsole darf keine relativen Koordinaten f�r den Standort verwenden.
+acf-minecraft.username_too_short = Fehler: Spielername zu kurz, er muss mindestens drei Buchstaben lang sein.
+acf-minecraft.is_not_a_valid_name = Fehler: <c2>{name}</c2> ist kein gültiger Spielername.
+acf-minecraft.multiple_players_match = Fehler: Es gibt mehrere Spieler deren Namen mit <c2>{search}</c2> <c3>({all})</c3> übereinstimmen, du musst genauer sein.
+acf-minecraft.no_player_found_server = Es wurde auf diesem Server kein mit <c2>{search}</c2> übereinstimmender Spieler gefunden.
+acf-minecraft.no_player_found_offline = Es wurde kein mit <c2>{search}</c2> übereinstimmender Spieler gefunden.
+acf-minecraft.no_player_found = Es wurde kein mit <c2>{search}</c2> übereinstimmender Spieler gefunden.
+acf-minecraft.location_please_specify_world = Fehler: Bitte die Welt angeben. Beispiel: <c2>world:x,y,z</c2>.
+acf-minecraft.location_please_specify_xyz = Fehler: Bitte x, y, z. Beispiel: <c2> Welt: x, y, z </ c2>.
```

Comment:
There is a "angeben" missing here, before the example.

Reply:
Oh yes, it's also missing in english.